src/main/webapp/feed.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Message Feed</title>
+<link rel="stylesheet" href="/css/main.css">
+<link rel="stylesheet" href="/css/user-page.css">
```

Comment:
So style/nitpicky thing that @acolwell may have better input on - it looks like you're using user-page.css as a stylesheet. Are you sure that you want to apply this style to your public feed page? Or should we create a separate css file which contains a style guide for the public feed page, even if it is similar to the user-page? 
I style these as questions because I don't know the answer, what you have might be great

Reply:
@nbaldy so I think it is ok to use it here. There is always this tradeoff of where to put things and how many style sheets to use. In this case it looks like @drewbern is trying to reuse the message-header and message-div classes in user-page.css. This makes sense to me for keeping messages looking consistent. The one area that makes me take pause is that user-page.css also has styles that target elements with specific IDs,  like "#about-me-input" which definitely feels user-page specific to me.

One could argue that if the .message-xxx styles are going to be useful on a lot more pages, it might make sense to move them to main.css. It is really up to you all about what convention you want to follow.

Reply:
The tutorial had user-page.css as the stylesheet, but thanks for pointing that out. I think we should create a separate css file for the public feed so we can build out a more robust website. I'll make user-page.css contents into a css file for the feed, and maybe a later PR can detail the formatting more based on how we want it to look.

Reply:
Saw @acolwell 's comment after I made the adjustment, but if we have more pages that need a common way to display messages, I like the idea of moving .message-xxx into main.css. Or maybe eventually making a separate feed-page.css that adjusts other feed specific features of feed.html. Although, I'm brand new to how .css files and .html files work, so suggestions and explanations are extremely appreciated! I've already gained some insight.

Reply:
Yeah I have absolutely no idea how anything works! My philosophy is to break it and start over like 10 times before getting to a decent result... and then I find something that I want to improve and the process starts all over. 

@acolwell explained them a little to me on my 1:1, if I remember correctly, then html is what the user sees, css is the "style" (so, for instance, you can have a mobile style and a desktop style), and javascript is how things interact with the user.... but that (if it's right) is about all I know so far

Reply:
I second the idea of moving things into main if they're used a lot, unless it breaks other stuff. Maybe check with everyone else, to make sure it won't break their stuff?

Reply:
> Yeah I have absolutely no idea how anything works! My philosophy is to break it and start over like 10 times before getting to a decent result... and then I find something that I want to improve and the process starts all over.
> 
:) This is basically how I work all the time. I usually don't have a great way to solve the problem or even really understand all the concerns until I've written it a few times. It is good practice and helps me learn. I also find it helps me not get too attached to one particular solution. Sometimes people get really attached to a piece of code because they spent so much time on it. The attachment is to the time spent instead of the actual solution which can cause all sorts of problems down the road sometime. Being willing to delete the code and try again is a good way to combat that.